code/modules/cargo/exports/materials.dm

```diff
@@ -13,17 +17,15 @@
 		return 0
 	if(!isitem(O))
 		return 0
+
 	var/obj/item/I = O
 	if(!(SSmaterials.GetMaterialRef(material_id) in I.custom_materials))
 		return 0
 
 	var/amount = I.custom_materials[SSmaterials.GetMaterialRef(material_id)]
 
-	if(istype(I, /obj/item/stack))
```

Comment:
yeah... this is the proper fix to https://github.com/BeeStation/BeeStation-Hornet/pull/10443

That fix was not correct.